src/groundStation/commandParser.py

```diff
@@ -34,19 +34,18 @@ def __init__(self):
     def parseInputValue(self, input):
         tokens = self._lexer(input)
         self._command = {}
-
+        
```

Comment:
Extra whitespace should be removed

Reply:
I don't think this is extra? Improves readability if there's empty lines between the if statements

Reply:
The newlines are fine, I just mean the spaces

Reply:
Ah I see